CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Fix leaking route params, as they may contain PII data ([#3487](https://github.com/getsentry/sentry-react-native/pull/3487))
```

Comment:
We could also link to the troubleshooting page https://docs.sentry.io/platforms/react-native/troubleshooting/#routing-transaction-data-contains-sensitive-information.